manifests/init.pp

```diff
@@ -73,6 +73,10 @@
 #   Can be either 'latest' or 'installed'; currently defaults to 'latest' for
 #   historical reasons. Default may change in a newer version.
 #
+# @param libkv Feature flag for libkv.
+#
+#   If set to true, it will enable the libkv backend for some functions.
```

Comment:
There could be a short definition of what libkv is, ie a distributed key value store.